prac_05/emails.py

```diff
@@ -0,0 +1,28 @@
+"""
+Write a program that stores users' emails (unique keys) and names (values) in a dictionary.
+Ask the user for their email until they enter a blank one.
+Use a separate function to get the name from the email
+"""
+
+
+def main():
+    email_directory = {}
```

Comment:
Name it closer to what goes in it? This dictionary is mapping... emails to names. So a good name might be email_to_name?